draft-06/links

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-06/hyper-schema#",
+    "$id": "http://json-schema.org/draft-06/links#",
```

Comment:
Because the links meta-schema is itself a hyperschema (and its self link also needs to be changed from "{+id}" to "{+$id}" or however we end up dealing with that).  And the links schema has its own id, of course.